pkg/controller/dataimportcron-controller.go

```diff
@@ -117,6 +109,14 @@ const (
 	digestDvNameSuffixLength    = 12
 	cronJobUIDSuffixLength      = 8
 	defaultImportsToKeepPerCron = 3
+
+	// ErrDataSourceAlreadyManaged provides a const to indicate DataSource already managed error
+	ErrDataSourceAlreadyManaged = "ErrDataSourceAlreadyManaged"
+	// MessageDataSourceAlreadyManaged provides a const to form DataSource already managed error message
+	MessageDataSourceAlreadyManaged = "DataSource %s is already managed by DataImportCron %s"
+
+	prometheusNsLabel       = "ns"
+	prometheusCronNameLabel = "cron_name"
```

Comment:
Why moved from const to var?

Reply:
I saw 2 const blocks so I merged them.
I saw that it appears here as a var too but for me locally it is as a const block. weird